src/components/paint-editor/paint-editor.jsx

```diff
@@ -205,7 +213,23 @@ class PaintEditorComponent extends React.Component {
                                 <StrokeWidthIndicatorComponent
                                     onUpdateSvg={this.props.onUpdateSvg}
                                 />
-                            </div>
+                            </InputGroup>
+                            <InputGroup className={classNames(styles.modDashedBorder, styles.modLabeledIconHeight)}>
+                                <LabeledIconButton
+                                    disabled
+                                    imgAlt="Copy Icon"
```

Comment:
We should define all of these in 'define messages' at some point.

Let's not use the word 'Icon' in the alt text

Reply:
I checked it out with a screenreader, and the current state is
- It reads the button and text on the button as separate focus items. Ideally it would be one item
- When it reads the button it seems to randomly pick between saying just "button" and the alt text

Obviously there's a lot to fix for screen readers so I wouldn't spend too much time on it

Reply:
Thanks! I just updated with the changes discussed